src/main/kotlin/de/michaelzinn/playerservices/Persistence.kt

```diff
@@ -0,0 +1,31 @@
+package de.michaelzinn.playerservices
+
+import de.michaelzinn.playerservices.data.PlayerServiceEntry
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import java.io.File
+
+data object Persistence {
```

Comment:
Furthermore, [persistence is already implemented](https://github.com/Michael-Zinn/playerservices/blob/d9fafaf1bfdfcbe27e99b0095a2aaf30a36c7cca/src/main/kotlin/de/michaelzinn/playerservices/PlayerServices.kt#L162). Like mentioned in the general comment, please rebase your work on top of the current state to avoid duplicating even more work.